packages/core/src/i18n.t.ts

```diff
@@ -1,14 +1,10 @@
+// Already listed, not sure what else to do
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Replace, Simplify, Trim, UnionToIntersection } from "type-fest"
```

Comment:
if you use webstorm and see warning from eslint from this rule that's probable because webstorm's eslint service should be restarted. It doesn't pick up changes in package.json automatically and continues to show error even if the package is already added. However this is not the case when you run eslint command directly.